cloudbuild.yaml

```diff
@@ -0,0 +1,43 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
+
+# Container to publish git repositories over HTTP.
+- name: 'gcr.io/cloud-builders/docker'
+  args: [
+      'build', '--tag', 'git-http-backend',
+      'https://github.com/v1bri/git-http-backend.git#git-daemon-package' ]
```

Comment:
What is this URL supposed to do?

Reply:
This is the URL to the (working) git HTTP server container. I had to make a small change to the original project and have an outstanding pull request. Added a TODO comment.

Reply:
What does the "#git-daemon-package" in the URL mean? It doesn't seem to be an anchor.

Reply:
It's Docker's URL format [1] to express fetching a particular branch.

[1] https://docs.docker.com/engine/reference/commandline/build/#git-repositories